test/jasmine/tests/click_test.js

```diff
@@ -180,7 +180,9 @@ describe('Test click interactions:', function() {
 
     describe('drag interactions', function() {
         beforeEach(function(done) {
-            Plotly.plot(gd, mockCopy.data, mockCopy.layout).then(done);
+            Plotly.plot(gd, mockCopy.data, mockCopy.layout).then(function() {
+                setTimeout(done, 200);
```

Comment:
> Shall I drop 04ef33e from this PR?

Yes please.

Those additional flags from Chrome would make a great addition, but they should be part of a different PR.